roles/cockpit/templates/listen.conf.j2

```diff
@@ -0,0 +1,4 @@
+[Socket]
+ListenStream=
+ListenStream=172.19.0.1:{{ cockpit_web_port }}
```

Comment:
You will need to make cockpit start after docker, in their systemd services, to listen on this.